datadog_checks_base/datadog_checks/utils/__init__.py

```diff
@@ -1,3 +1,12 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+
+try:
+    import datadog_agent
+    if datadog_agent.get_config('integration_tracing'):
+        from ddtrace import patch
+        patch(requests=True)
```

Comment:
Do we plan to make this configurable? Or we just add the modules we want auto-instrumentation for as long as we need them? (to be clear, potentially having in the future something like `patch(requests=True, psycopg=True, redis=True)` or `patch_all()` directly)

Reply:
Thats what I had in mind with this. We can always make it configurable later from datadog.yaml by having something like:
```
integrations_tracing: 
    requests: True
    psycopg: True
```

But for now I think this is OK to just include the things we need globally here.